pkg/skaffold/constants/constants.go

```diff
@@ -46,7 +46,7 @@ const (
 
 	DefaultKustomizationPath = "."
 
-	DefaultKanikoImage             = "gcr.io/kaniko-project/executor:v0.4.0@sha256:0bbaa4859eec9796d32ab45e6c1627562dbc7796e40450295b9604cd3f4197af"
+	DefaultKanikoImage             = "gcr.io/kaniko-project/executor@sha256:434bbb1d998ba1bd8ebc04c90d93afa859fd5c7ff93326bca9f6e7da0d6277ff"
```

Comment:
Can you leave the version here, in addition to the sha256? It makes it easier to find which version we are using.

Reply:
kaniko actually hasn't released a version with the new caching stuff yet -- I was going to come back to this and add the new version in once that happened.

Reply:
I'll open an issue for it!